server/tests/unit/middleware/validation/date.test.ts

```diff
@@ -87,7 +87,9 @@ describe('Date validation', () => {
   });
 
   describe('Listing with no deadline', () => {
-    const {deadline, ...listingWithNoDeadline} = validListing;
+    /* eslint-disable @typescript-eslint/no-unused-vars */
+    const {deadline: _, ...listingWithNoDeadline} = validListing;
+    /* eslint-enable @typescript-eslint/no-unused-vars */
```

Comment:
If only disabling for 1 line can consider using `eslint-disable-next-line`! But for this I think what we can do is just add this as a general rule in `eslintrc.json` like I have done here https://github.com/googleinterns/gpay-group-buy/blob/master/web/.eslintrc.json#L17

Reply:
@hzxcaryn Ohh! thanks will just disable the rule for _ in general.